pulp_ansible/app/tasks/collections.py

```diff
@@ -360,6 +360,12 @@ def __init__(self, remote, repository, optimize):
         self.optimize = optimize
         self.collection_info = parse_collections_requirements_file(remote.requirements_file)
         self.deprecations = Q()
+        self.add_dependents = False
+        if self.collection_info and self.remote.sync_dependencies:
+            self.add_dependents = True
+            self.collection_queue = asyncio.Queue()
```

Comment:
I may be missing something here, but I imagined we didn't need a dedicated codepath to handle this case differently from our existing sync code. Can't we take new requirements and spawn new coroutines to handle them as if they came from the original requirements file?